src/components/dropdown/DropDown.js

```diff
@@ -1,66 +1,82 @@
+        // dtodo: вызвать функцию из onSelect с аргументом value
+        onSelect(value)
 
+        // dtodo: закрыть дропдаун, вызвав this.setState({ isOpen: false })
+        this.setState({isOpen: false})
+    };
 
-    // todo: вызвать функцию из onSelect с аргументом value
+    render() {
+        // tdodo:
+        //  достать из this.props массив опций options
+        //  он должен быть по умолчанию пуст, т.е. = [] (дефолтный параметр при деструктуризации) и
+        //  достать также в пропсах selectedOption (выбранная в данный момент опция)
+        const {options = [], selectedOption} = this.props
+        // dtodo:
+        //  достать isOpen из this.state
+        const {isOpen} = this.state
 
-
-    // todo: закрыть дропдаун, вызвав this.setState({ isOpen: false })
-  };
-
-  render() {
-    // todo:
-    //  достать из this.props массив опций options
-    //  он должен быть по умолчанию пуст, т.е. = [] (дефолтный параметр при деструктуризации) и
-    //  достать также в пропсах selectedOption (выбранная в данный момент опция)
-
-    // todo:
-    //  достать isOpen из this.state
-
-    return (
-      <div className="may-drop-down dropdown">
-        {/* todo: показать в строке 45 выбранную опцию selectedOption  и передать в onClick this.toggle
+        return (
+            <div className="may-drop-down dropdown">
+                {/* tdodo: показать в строке 45 выбранную опцию selectedOption  и передать в onClick this.toggle
              (нажатие на этот тег должно открывать/закрывать дропдаун)
         */}
-        <div className="dropdown-toggle">{/* тут будет выбранная опция*/}</div>
+                <div className="dropdown-toggle" onClick={this.toggle}>{selectedOption}{/* тут будет выбранная опция*/}</div>
 
-        {
-          // todo:
-          //  если значение isOpen в this.state = true показываем этот блок ниже (использовать &&)
-          <div className="may-drop-down-options-wrapper dropdown-menu show">
-            {/*
-          todo:
+                {
+                    // toddo:
+                    //  если значение isOpen в this.state = true показываем этот блок ниже (использовать &&)
+                 isOpen &&  <div className="may-drop-down-options-wrapper dropdown-menu show">
+                        {/*
+          toddo:
             //  рендерим список опций с помощью options.map(option => ....
             //  опция должна быть тегом div и иметь такие пропсы:
             //  должен быть className = "may-drop-down-options-wrapper-option dropdown-item"
             //  key = значению option
             //  атрибут data-value =  значению option
             //  в event listener onClick положить значение this.onOptionSelect
             //  внутри тега показать {option} (как children)
-          */}
-          </div>
-        }
-      </div>
-    );
-  }
+
+          */
+                        options.map(options => {
```

Comment:
it's better to name the argument of an iterator as `option` or `item` because the could be collisions of names